doc/example/example_ordinal.ipynb

```diff
@@ -0,0 +1,1047 @@
+   "execution_count": 65,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import logging\n",
+    "\n",
+    "import amici\n",
+    "import numpy as np\n",
+    "import petab\n",
+    "\n",
+    "import pypesto\n",
+    "import pypesto.logging\n",
+    "import pypesto.optimize as optimize\n",
+    "import pypesto.petab\n",
+    "from pypesto.hierarchical.optimal_scaling_approach.optimal_scaling_solver import (\n",
+    "    OptimalScalingInnerSolver,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 66,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "petab_folder = './example_ordinal/'\n",
+    "yaml_file = 'example_ordinal.yaml'\n",
+    "\n",
+    "petab_problem = petab.Problem.from_yaml(petab_folder + yaml_file)\n",
+    "\n",
+    "# To allow for optimization with ordinal measurements, set ordinal=True, when creating the importer\n",
+    "importer = pypesto.petab.PetabImporter(petab_problem, ordinal=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 67,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "objective = importer.create_objective()\n",
+    "problem = importer.create_problem(objective)\n",
+    "\n",
+    "engine = pypesto.engine.SingleCoreEngine()\n",
+    "\n",
+    "optimizer = optimize.ScipyOptimizer(\n",
+    "    method='Nelder-Mead',\n",
+    "    options={'disp': True, 'maxiter': 500, 'maxfev': 500, 'fatol': 1e-10},\n",
+    ")\n",
+    "n_starts = 10\n",
+    "np.random.seed(n_starts)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run optimization using optimal scaling approach"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Different options can be used for the optimal scaling approach:\n",
+    "- method: `standard` / `reduced`\n",
+    "- reparameterized: `True` / `False`\n",
+    "- intervalConstraints: `max` / `max-min`\n",
+    "- minGap: Any float value\n",
+    "\n",
+    "It is recommended to use the reduced method with reparameterization as it is the most efficient and robust choice.\n",
+    "\n",
+    "When no options are provided, the default is the reduced and reparameterized formulation with max as interval constraint and `minGap=1e-10`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Run optimization using the reduced and reparameterized approach"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 68,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      " 10%|█         | 1/10 [00:02<00:25,  2.84s/it]"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Optimization terminated successfully.\n",
+      "         Current function value: 0.000574\n",
+      "         Iterations: 146\n",
+      "         Function evaluations: 271\n"
```

Comment:
Fine to output this data in one cell if it's somehow especially useful to a user using your work, but otherwise would be good to disable by default and/or mute in other cells of the notebook

Reply:
Will mute it, it's too much output for a notebook.